pyuploadcare/api/api.py

```diff
@@ -520,9 +523,11 @@ def _get_request_data(
     def execute(
         self,
         file_uuid: Union[UUID, str],
-        addon_name: AddonLabels,
+        addon_name: Union[AddonLabels, str],
         params: Optional[AddonExecutionParams] = None,
     ) -> responses.AddonExecuteResponse:
+        if isinstance(addon_name, AddonLabels):
+            addon_name = addon_name.value
```

Comment:
Previously, when we call `.execute(..., AddonLabels.CLAM_AV)`, POST request would be sent to `https://api.uploadcare.com/addons/AddonLabels.REMOVE_BG/execute/` — that made no sense.

[pyuploadcare-example](https://github.com/uploadcare/pyuploadcare-example/) worked because for some reason  it uses strings (like `uc_clamav_virus_scan`) instead of the enum. Which shouldn't pass mypy validation.

Now both variants are acceptable.